docker-environment-driver/pom.xml

```diff
@@ -1,85 +1,249 @@
-    </dependency>
-        
-    <dependency>
-      <groupId>org.apache.jclouds.labs</groupId>
-      <artifactId>docker</artifactId>
-    </dependency>
-    
-    <dependency>
-      <groupId>org.apache.jclouds</groupId>
-      <artifactId>jclouds-compute</artifactId>
-    </dependency>
-    
-    <dependency>
-      <groupId>org.apache.jclouds.driver</groupId>
-      <artifactId>jclouds-sshj</artifactId>
-    </dependency>
-    
-    <dependency>
-      <groupId>org.apache.jclouds.driver</groupId>
-      <artifactId>jclouds-slf4j</artifactId>
-    </dependency>
-    
-    <dependency>
-      <groupId>com.google.guava</groupId>
-      <artifactId>guava</artifactId>
-    </dependency>
-    
-  </dependencies>
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
 
+	<parent>
+		<groupId>org.jboss.pnc</groupId>
+		<artifactId>parent</artifactId>
+		<version>1.0-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>docker-environment-driver</artifactId>
+
+	<properties>
+		<java.util.logging.manager>org.jboss.logmanager.LogManager</java.util.logging.manager>
+		<jboss.version>8.2.0.Final</jboss.version>
+		<app.server>wildfly</app.server>
+		<test.server.unpack.dir>${project.build.directory}</test.server.unpack.dir>
+		<test.server.build.dir>${test.server.unpack.dir}/${app.server}-${jboss.version}</test.server.build.dir>
+	</properties>
+
+	<repositories>
+		<repository>
+			<id>jboss-public-repository-group</id>
+			<url>https://repository.jboss.org/nexus/content/groups/public-jboss/</url>
+		</repository>
+	</repositories>
+
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>org.wildfly</groupId>
+				<artifactId>wildfly-arquillian-container-embedded</artifactId>
+				<version>${jboss.version}</version>
+				<scope>test</scope>
+			</dependency>
+			<dependency>
+				<groupId>org.wildfly</groupId>
+				<artifactId>wildfly-embedded</artifactId>
+				<version>${jboss.version}</version>
+				<scope>test</scope>
+			</dependency>
+			<dependency>
+				<groupId>org.jboss.shrinkwrap.resolver</groupId>
+				<artifactId>shrinkwrap-resolver-bom</artifactId>
+				<version>2.1.1</version>
+				<scope>import</scope>
+				<type>pom</type>
+			</dependency>
+		</dependencies>
+	</dependencyManagement>
+
+	<dependencies>
+
+		<!-- Project dependencies -->
+		<dependency>
+			<groupId>org.jboss.pnc</groupId>
+			<artifactId>pnc-model</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.jboss.pnc</groupId>
+			<artifactId>common</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.jboss.pnc</groupId>
+			<artifactId>pnc-spi</artifactId>
+		</dependency>
+
+		<!-- Remote dependencies -->
+		<dependency>
+			<groupId>org.jboss.spec</groupId>
+			<artifactId>jboss-javaee-6.0</artifactId>
+			<type>pom</type>
+			<scope>provided</scope>
+		</dependency>
+
+		<dependency>
+			<groupId>javax.enterprise</groupId>
```

Comment:
Since we import JEE6 api - do we need those APIs? They all should be included in JEE6 spec.